README.md

```diff
@@ -0,0 +1,19 @@
+# Masonite Azure Driver
+
+<p align="center">
+  <img src="https://i.imgur.com/rEXcoMn.png" width="160px">
+</p>
+
+## About Masonite Azure Driver
+
+Store files into Microsoft Azure Server either with a file name or with a pre-pend file name
```

Comment:
let's changed the `pre-pend` word to `prefixed`

Reply:
done!